### ISSUE TITLE
Error while restoring F# project on 2.0.0-preview2

### ISSUE THREAD

**Issue by lvicentesanchez** [dotnet/cli#6862]
## Steps to reproduce
Using the sdk docker image `microsoft/dotnet-nightly:2.0.0-preview2-sdk-jessie` run `dotnet new console -lang f# -n hello`

## Expected  behavior

The project should be created and restored properly

## Actual behavior

```
root@b6ab840780e5:/tmp# dotnet new console -lang f# -n hello 
The template "Console Application" was created successfully.

Processing post-creation actions...
Running 'dotnet restore' on hello/hello.fsproj...
Restore failed.
Output from command:
/usr/share/dotnet/sdk/2.0.0-preview2-006215/FSharp/Microsoft.FSharp.NetSdk.targets(44,3): error MSB4019: The imported project "/usr/share/dotnet/sdk/2.0.0-preview2-006215/FSharp/Microsoft.FSharp.targets" was not found. Confirm that the path in the <Import> declaration is correct, and that the file exists on disk. [/tmp/hello/hello.fsproj]






Post action failed.
Description: Restore NuGet packages required by this project.
Manual instructions: Run 'dotnet restore'
```

Content of the `/usr/share/dotnet/sdk/2.0.0-preview2-006215/FSharp` folder:

```
root@b6ab840780e5:/usr/share/dotnet/sdk/2.0.0-preview2-006215/FSharp# ls -la
total 29384
drwxr-xr-x  3 1002 sudo     4096 Jun  1 16:57 .
drwxr-xr-x 23 1002 sudo    12288 Jun  1 16:57 ..
-rw-r--r--  1 1002 sudo   128000 Jun  1 16:45 FSharp.Build.dll
-rw-r--r--  1 1002 sudo 23449600 Jun  1 16:45 FSharp.Compiler.dll
-rw-r--r--  1 1002 sudo  4410368 Jun  1 16:45 FSharp.Core.dll
-rw-r--r--  1 1002 sudo   430791 May 11 06:39 FSharp.Core.optdata
-rw-r--r--  1 1002 sudo   570999 May 11 06:39 FSharp.Core.sigdata
-rw-r--r--  1 1002 sudo   718843 May 11 06:39 FSharp.Core.xml
-rw-r--r--  1 1002 sudo   203264 Jun  1 16:45 Microsoft.DiaSymReader.PortablePdb.dll
-rw-r--r--  1 1002 sudo    17408 Jun  1 16:45 Microsoft.DiaSymReader.dll
-rw-r--r--  1 1002 sudo     2093 May 11 06:39 Microsoft.FSharp.NetSdk.props
-rw-r--r--  1 1002 sudo     6290 May 11 06:39 Microsoft.FSharp.NetSdk.targets
-rw-r--r--  1 1002 sudo    16865 May 11 06:39 Microsoft.FSharp.Targets
-rw-r--r--  1 1002 sudo     2125 May 11 06:39 Microsoft.Portable.FSharp.Targets
-rw-r--r--  1 1002 sudo      229 Jun  1 16:30 RunFsc.cmd
-rw-r--r--  1 1002 sudo      661 Jun  1 16:30 RunFsc.sh
-rw-r--r--  1 1002 sudo      490 May 11 06:39 default.win32manifest
-rw-r--r--  1 1002 sudo    54868 Jun  1 16:44 fsc.deps.json
-rw-r--r--  1 1002 sudo    12288 Jun  1 16:45 fsc.exe
-rw-r--r--  1 1002 sudo      164 Jun  1 16:44 fsc.runtimeconfig.json
drwxr-xr-x  3 1002 sudo     4096 Jun  1 16:57 runtimes
```

## Environment data
`dotnet --info` output:

```
root@b6ab840780e5:/tmp# dotnet --info
.NET Command Line Tools (2.0.0-preview2-006215)

Product Information:
 Version:            2.0.0-preview2-006215
 Commit SHA-1 hash:  e67962b0de

Runtime Environment:
 OS Name:     debian
 OS Version:  8
 OS Platform: Linux
 RID:         debian.8-x64
 Base Path:   /usr/share/dotnet/sdk/2.0.0-preview2-006215/

Microsoft .NET Core Shared Framework Host

  Version  : 2.0.0-preview2-25324-03
  Build    : b6c9f8ad69dc3edd601e288198de27975b514306
```

**Comment by livarcocc** [comment]
@KevinRansom

**Comment by ellismg** [comment]
Note the casing of `.Targets` in the Microsoft.FSharp.Targets filename, vs what the import statement says.  On case specific file systems, this is going to be problematic.

**Comment by livarcocc** [comment]
I am surprised this is not causing CI to tail.

**Comment by livarcocc** [comment]
@cartermp Who can look at this in @KevinRansom's absence?

**Comment by cartermp** [comment]
@brettfo is looking at this in his absence.  Though I thought @KevinRansom had identified issues with casing/spelling and fixed them...

**Comment by livarcocc** [comment]
This has been fixed now in the release/2.0.0-preview2 branch. We will merge it back into release/2.0.0 soon. This was fixed by taking a new fsharp compiler.
